en/user_guide/node_configuration.rst

```diff
@@ -0,0 +1,105 @@
+Node configuration
+==================
+
+Here are the available options in the node configuration. Some of them are required
+in order to build a node and must therefore be provided.
+
+Page (required)
+---------------
+
+The page attribute is the name of the node, it is also used to build the url pattern.
+
+Url pattern (required)
+----------------------
+
+The url pattern attribute is generated automatically with the name of the page,
+it represents the pattern that will be the URL of the page on the website.
```

Comment:
As this is a user guide, not a technical guide, maybe an exemple of what is a pattern could help contributors with a non technical profile.